sensor/docs/tutorials/tag_based_pnp_calibrator.md

```diff
@@ -0,0 +1,140 @@
+# tag_based_pnp_calibrator
+
+In this tutorial, we will present a hands-on tutorial of the `tag_based_pnp_calibrator`. Although we provide pre-recorded rosbag, the flow of the tutorial is meant to show the user the steps they must perform in their own use cases with live sensors.
+
+General documentation regarding this calibrator can be found [here](../../tag_based_pnp_calibrator/README.md).
+
+## Setup
+
+This tutorial assumes that the user has already built the calibration tools.
+Installation instructions can be found [here](../../README.md)
+
+## Data preparation
+
+Please download the data (rosbag) from [here](https://drive.google.com/drive/folders/1gFrrchW9mWM1huWMYuJ0nWB2n1BfCJag).
+
+The rosabg includes four different topics including `camera_info`, `image_rect_color/compressed`, `pointcloud_raw`, and `tf_static`.
```

Comment:
The provided rosbag

You can not say "including four topics including" and then list the four topics

Reply:
Thanks! fixed in https://github.com/tier4/CalibrationTools/commit/32dd61431d3beb3d84ecb085cd246042de8efb09

Reply:
In your new version I think you are missing a ":"

Reply:
sovled in https://github.com/tier4/CalibrationTools/pull/164/commits/62eabe1bc9a93c12cc6948924c8adeb58a36646d